libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -604,6 +604,7 @@ MAV_MISSION_RESULT AP_Mission::mavlink_int_to_mission_cmd(const mavlink_mission_
     case MAV_CMD_NAV_LAND:                              // MAV ID: 21
         copy_location = true;
         cmd.p1 = packet.param1;                         // abort target altitude(m)  (plane only)
+        cmd.content.location.flags.loiter_ccw = is_negative(packet.param4); // yaw direction, (plane deepstall only)
```

Comment:
uh oh.. This PR was looking perfect until I saw this. You so sneaky!

Reply:
Whats wrong with it? Param4 has an official meaning for yaw in the MAVLink spec :) If someone else wants to actually use the value we could stick it in a struct, but till then the flags field felt great :)

Reply:
hmm.. interesting. I didn't notice that at all! Thought you were trying to do something funny there. Although it's used out of spec still, a comment in common.xml would be required...
https://github.com/mavlink/mavlink/blob/master/message_definitions/v1.0/common.xml#L703

Reply:
its a bit strange, but useful, so I think we can do this